tests/SAML2/XML/saml/AuthenticatingAuthorityTest.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SAML2\XML\saml;
+
+use SAML2\Constants;
+use SAML2\DOMDocumentFactory;
+use SAML2\Utils;
+
+/**
+ * Class \SAML2\XML\saml\AuthenticatingAuthorityTest
+ */
+class AuthenticatingAuthorityTest extends \PHPUnit\Framework\TestCase
+{
+    /**
+     * @return void
+     */
+    public function testMarshalling(): void
```

Comment:
With the new `AbstractConvertable` class, we can now simplify tests like this by casting the object to a string and comparing with a string representing the XML document.